pkg/planner/core/expression_test.go

```diff
@@ -40,22 +40,40 @@ func parseExpr(t *testing.T, expr string) ast.ExprNode {
 	return stmt.Fields.Fields[0].Expr
 }
 
-func buildExpr(t *testing.T, ctx sessionctx.Context, exprNode any, opts ...expression.BuildOption) (expression.Expression, error) {
+func buildExpr(t *testing.T, ctx sessionctx.Context, exprNode any, opts ...expression.BuildOption) (expression.Expression, expression.Expression, error) {
```

Comment:
(Minor comment, since it is in a test...)
Why having a function that depending on input returns an expression to either of the two resulting values, depending on the input type?
Would it not be more clear to have two separate functions? Or to simply return only one result expression, independently of the input?
Also where is the second return value used?

Reply:
It is to check the `ParseSimpleExpr` will build the same expression with parse and rewrite. However, yes, it is a little difficult to understand, I'll change it.